src/pages/ImportData.js

```diff
@@ -392,7 +393,28 @@ const ImportData = () => {
     };
   };
 
-  const importTdrExport = (namespace, name) => {
+  const loadWdsUrl = useCallback((workspaceId) => {
+    return Ajax()
+      .Apps.listAppsV2(workspaceId)
+      .then(resolveWdsUrl)
+      .then((url) => {
+        return url;
+      });
+  }, []);
+
+  const importTdrExport = (workspace) => {
+    // For new workspaces, cloudPlatform is blank
+    if (workspace.cloudPlatform === 'Azure' || workspace.googleProject === '') {
```

Comment:
I would like to prevent import into newly created azure workspaces, as at the moment it will fail - working with creating a new workspace for import will likely happen in a follow-on ticket.  However the `NewWorkspaceModal` doesn't seem to have an easy way to prevent certain billing projects and I'm not sure how much refactoring is worth it for something temporary.

Reply:
Couple questions:
1. Is googleProject ever going to be undefined rather than `''`?
2.  Is there any case where the cloud platform is 'Azure' and there isn't also an empty `googleProject field? If there isn't, did you think about removing the first half of this conditional and just checking that it is not GCP? 
3. And then so I understand, we are calling rawls still for the case where it is a new workspace or a google workspace? And that's because WDS doesn't yet support importing data in those cases?

Reply:
1. I didn't see any examples of that, but good point!
2. I suppose I could only use the second half of the conditional, but at least specifically looking for 'Azure' makes the intention clear.
3. Using WDS for a new workspace is complicated because we need to wait for WDS to boot up.  Dealing with that is left for later work, so I'm leaving the behavior unchanged, which means calling rawls - which will fail for an Azure workspace.

Reply:
if `workspace.googleProject === ''` indicates a new workspace, then wouldn't this conditional break import into new Google workspaces?

Reply:
ignore my previous comment about `workspace.googleProject === ''` indicating a new workspace; I misread the code comment. That said, could you replace the conditional with:
```javascript
import { isAzureWorkspace } from 'src/libs/workspace-utils';

…

if (isAzureWorkspace(workspace)) {

```
in other words, do you need the check against `workspace.googleProject`?

Reply:
`isAzureWorkspace` in `src/libs/workspace-utils`, is checking the cloudPlatform, which I've discovered isn't present when creating a new workspace.  But since new workspaces will fail for Azure anyway, I suppose we can go ahead and use it - both google and azure new workspaces will fail the conditional, but that won't change any behavior at this time.  It's something we'll need to keep in mind for when we address new azure workspaces.

Reply:
gotcha. I now (finally) understand what all the checks here are meant to do. Also agree that in the new-workspace case (blank `cloudPlatform`), it doesn't matter. Thank you for bearing with me!